### DISCUSSION TITLE
Fluxstore Manager - Support for Custom WooCommerce Order Status

### DISCUSSION THREAD

**Discussion by anshverma14** [inspireui/support#5172] [New Feature/Ideas Requests]
Please enable the support for custom woocommerce order status in fluxstore manager, Its Highly required for smoot functioning of  order process via a mobile.

**Comment by tryabp** [comment]
yes, WooCommerce Order Status Manager (https://woocommerce.com/products/woocommerce-order-status-manager/) works best and FluxStore team should seriously consider it.
